vm/actor/tests/common/mod.rs

```diff
@@ -512,11 +565,73 @@ impl<BS: BlockStore> Runtime<BS> for MockRuntime<'_, BS> {
         todo!("implement me???")
     }
 
+    fn total_fil_circ_supply(&self) -> Result<TokenAmount, ActorError> {
+        unimplemented!();
+    }
+
     fn syscalls(&self) -> &dyn Syscalls {
-        unimplemented!()
+        self
     }
+}
 
-    fn total_fil_circ_supply(&self) -> Result<TokenAmount, ActorError> {
-        unimplemented!()
+impl<BS> Syscalls for MockRuntime<'_, BS>
+where
+    BS: BlockStore,
+{
+    fn verify_signature(
+        &self,
+        signature: &Signature,
+        signer: &Address,
+        plaintext: &[u8],
+    ) -> Result<(), Box<dyn StdError>> {
+        let op_exp = self.expect_verify_sig.replace(Option::None);
+
+        if let Some(exp) = op_exp {
+            if exp.sig == *signature && exp.signer == *signer && (exp.plaintext[..]) == *plaintext {
```

Comment:
```suggestion
            if exp.sig == *signature && exp.signer == *signer && &exp.plaintext[..] == plaintext {
```
Just a bit weird to read as is, could be more explicit with `.as_slice()` too for comparison